istioctl/pkg/waypoint/waypoint.go

```diff
@@ -310,6 +348,7 @@ func Cmd(ctx cli.Context) *cobra.Command {
 	}
 
 	waypointApplyCmd.PersistentFlags().StringVarP(&revision, "revision", "r", "", "The revision to label the waypoint with")
+	waypointApplyCmd.PersistentFlags().BoolVarP(&waitReady, "wait-ready", "w", false, "Wait for the waypoint to be ready")
```

Comment:
Sounds reasonable to me, I don't think the user will want to wait for another condition that is not "ready".
Therefore, it makes sense to use --wait instead of --wait-ready.
I will change it in the next commit.